compiler/rustc_hir_analysis/src/astconv/mod.rs

```diff
@@ -2422,6 +2422,10 @@ impl<'o, 'tcx> dyn AstConv<'tcx> + 'o {
             hir::TyKind::Tup(fields) => {
                 Ty::new_tup_from_iter(tcx, fields.iter().map(|t| self.ast_ty_to_ty(t)))
             }
+            hir::TyKind::AnonAdt(item_id) => {
+                let adt_def = crate::collect::adt_def(tcx, item_id.owner_id.def_id);
```

Comment:
Why not `tcx.adt_def(item_id.owner_id)`?

Reply:
Will fix it later. It was because I didn't know `adt_def` is a query.